src/main/resources/META-INF/rewrite/junit5-assertions-to-assertj.yml

```diff
@@ -28,9 +28,3 @@ recipeList:
   - org.openrewrite.java.testing.junitassertj.JUnitFailToAssertJFail
   - org.openrewrite.java.UseStaticImport:
       methodPattern: "org.assertj.core.api.Assertions *(..)"
-  - org.openrewrite.java.ChangeMethodTargetToStatic
-      methodPattern: "org.assertj.core.api.AssertionsForClassTypes.assertThat *(..)"
-      fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions.assertThat"
-  - org.openrewrite.java.ChangeMethodTargetToStatic
-    methodPattern: "org.assertj.core.api.AssertionsForInterfaceTypes.assertThat *(..)"
-    fullyQualifiedTargetTypeName: "org.assertj.core.api.Assertions.assertThat"
```

Comment:
Normally I would say that it is sensible to separate a "best practices" recipe from a "migration" recipe. Migrations you run once, best practices you run continually. 
But, as I alluded to in the other pull request, assertj's migration is also simultaneously a best practices recipe since it's _super easy_ to reintroduce JUnit assertions even after running the migration once. 

So I don't really care if these ChangeMethodTargetToStatics are put in their own recipe as well, but the overall recipe should include that recipe if they are

Reply:
There isn't currently an 'overall' / "opinionated default" recipe for assertj, should I make one and hang these off that? The closest thing is "migrate junit5 assertions to assertj assertions". I think that'd solve issues, honestly. That's be like the loose conceptual equivalent of AutoFormat (sort of) where it's a one-stop shop of recipes, which are composed of sub-recipes.

I can do that and call it something like `DefaultAssertJ`?